src/SplQueue/PHP81Implementation.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib\SplQueue;
+
+use SplQueue;
+use UnexpectedValueException;
+
+use function serialize;
+use function unserialize;
+
+/**
+ * Serializable version of SplQueue
+ */
+class PHP81Implementation extends SplQueue
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API